src/content/sidebar/sidebar-wrapper.tsx

```diff
@@ -1,59 +1,130 @@
 import * as browser from "webextension-polyfill"
 
-import {css} from "@emotion/react"
-import {slide as Slider} from 'react-burger-menu'
-import {useEffect, useState} from "react"
+import {useEffect, useState, useRef} from "react"
 import {useTabLocalState} from "../../core/react"
 
 export const SidebarWrapper = () => {
+    const ref = useRef()
+
     const [isOpen, setOpen] = useTabLocalState("sidebarOpen", false)
+
+    // the iframe should be rendered as soon as `isOpen` is true but
+    // when `isOpen` becomes `false` the iframe should only dissapear
+    // after the hiding animation is completed 
+    const [showIFrame, setShowIFrame] = useState(isOpen)
```

Comment:
Imo it's not worth to have additional logic/complexity for this?

Reply:
I think it looks whacky without

Reply:
was this one of the things handled by the burger before, or was it "whacky" before too?